[Bug]: Unable to build netdata for v2.5.0

### Bug description

Error when updating netdata to v2.5.0. Here is the command: `wget -O /tmp/netdata-kickstart.sh https://get.netdata.cloud/kickstart.sh && sh /tmp/netdata-kickstart.sh --build-only --install-prefix /data2
`. I used this command a few times before without any issues.
```
/usr/bin/ld: /usr/bin/ld: DWARF error: could not find abbrev number 210
/tmp/ccTqxcki.ltrans25.ltrans.o: in function `metadata_event_loop':
<artificial>:(.text+0x23b58): undefined reference to `uv_sleep'
/usr/bin/ld: /usr/bin/ld: DWARF error: could not find abbrev number 6917
/tmp/ccTqxcki.ltrans31.ltrans.o: in function `create_uv_thread':
<artificial>:(.text+0x5f08): undefined reference to `uv_sleep'
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/netdata.dir/build.make:6502: netdata] Error 1
make[2]: Leaving directory '/tmp/netdata-updater-DPRDgJkZ6y/netdata-v2.5.0/build'
make[1]: *** [CMakeFiles/Makefile2:1812: CMakeFiles/netdata.dir/all] Error 2
make[1]: Leaving directory '/tmp/netdata-updater-DPRDgJkZ6y/netdata-v2.5.0/build'
make: *** [Makefile:156: all] Error 2
 FAILED  ''

 ABORTED  Failed to build Netdata.

Tue 06 May 2025 08:37:50 AM CDT : FATAL: netdata-updater.sh: FAILED TO UPDATE NETDATA:  Failed to rebuild existing netdata install: Failed to build Netdata.
 FAILED

The following non-fatal warnings or errors were encountered:

  -
  - Command '/usr/local/bin/cmake --build ./build/ --parallel 4 -- VERBOSE=1' failed with exit code 2.
  - Failed to build Netdata.
  - Failed to rebuild existing netdata install: Failed to build Netdata.

 ABORTED  Failed to update existing Netdata install at /data2/netdata: Failed to rebuild existing netdata install: Failed to build Netdata.

```

### Expected behavior

Should just work

### Steps to reproduce

1. Just run `wget -O /tmp/netdata-kickstart.sh https://get.netdata.cloud/kickstart.sh && sh /tmp/netdata-kickstart.sh --build-only --install-prefix /data2`
2.
3.
...


### Installation method

kickstart.sh

### System info

```shell
Linux serenity 4.19.0-27-amd64 #1 SMP Debian 4.19.316-1 (2024-06-25) x86_64 GNU/Linux
/etc/os-release:PRETTY_NAME="Debian GNU/Linux 10 (buster)"
/etc/os-release:NAME="Debian GNU/Linux"
/etc/os-release:VERSION_ID="10"
/etc/os-release:VERSION="10 (buster)"
/etc/os-release:VERSION_CODENAME=buster
/etc/os-release:ID=debian
```

### Netdata build info

```shell
After failure, no build info anymore
```

### Additional info

_No response_

--- a/src/daemon/libuv_workers.c
+++ b/src/daemon/libuv_workers.c
@@ -122,8 +122,7 @@ int create_uv_thread(uv_thread_t *thread, uv_thread_cb thread_func, void *arg, i
         err = uv_thread_create(thread, thread_func, arg);
         if (err == 0)
             break;
-
-        uv_sleep(MAX_THREAD_CREATE_WAIT_MS);
+        sleep_usec(MAX_THREAD_CREATE_WAIT_MS * USEC_PER_MS);
     } while (err == UV_EAGAIN && ++(*retries) < MAX_THREAD_CREATE_RETRIES);
 
     return err;

--- a/src/database/sqlite/sqlite_metadata.c
+++ b/src/database/sqlite/sqlite_metadata.c
@@ -2681,7 +2681,7 @@ static void metadata_event_loop(void *arg)
     while ((config->metadata_running || config->ctx_load_running) && --loop_count) {
         if (!uv_run(loop, UV_RUN_NOWAIT))
             break;  // No pending callbacks
-        uv_sleep(SHUTDOWN_SLEEP_INTERVAL_MS);
+        sleep_usec(SHUTDOWN_SLEEP_INTERVAL_MS * USEC_PER_MS);
     }
 
     (void)uv_loop_close(loop);

